awx/main/tasks.py

```diff
@@ -688,6 +688,13 @@ def get_path_to(self, *args):
         '''
         return os.path.abspath(os.path.join(os.path.dirname(__file__), *args))
 
+    def get_path_to_ansible(self, instance, executable='ansible-playbook', **kwargs):
+        venv_path = getattr(instance, 'ansible_virtualenv_path', settings.ANSIBLE_VENV_PATH)
+        venv_exe = os.path.join(venv_path, 'bin', executable)
+        if os.path.exists(venv_exe):
+            return venv_exe
+        return shutil.which(executable)
```

Comment:
ok, this makes sense as the fallback. Heaven help us that people don't install Ansible in the AWX venv.

Reply:
Yea, I think "look for the executable in the venv first", and we tell people to install ansible for custom venvs w/ a Python that doesn't match your system Python.

Reply:
So for distributions where py2 is the default Python, if you want a py3 venv, you have to make it and install ansible into it.

For distributions where py3 is the default Python, if you want a py2 venv, you have to make it and install ansible into it.